.circleci/config.yml

```diff
@@ -52,8 +52,17 @@ commands:
           curl -s --fail --location https://dl.google.com/go/go<<parameters.go_version>>.<<parameters.go_os>>-<<parameters.go_arch>>.tar.gz | tar -xz -C $HOME
           echo 'export PATH=$HOME/go/bin:$PATH' >> $BASH_ENV
           # if GOPATH is not set, `golang-ci` fails with an obscure message
-          # "ERRO Running error: context loading failed: failed to load program with go/packages: could not determine GOARCH and Go compiler"
+          # "ERROR Running error: context loading failed: failed to load program with go/packages: could not determine GOARCH and Go compiler"
           echo 'export GOPATH=$HOME/go' >> $BASH_ENV
+    - run:
+        name: "Set DNS server for docker daemon"
+        command: |
+          cat \<<'EOF' | sudo tee /etc/docker/daemon.json
+          {
+            "dns": ["8.8.8.8", "1.1.1.1"]
```

Comment:
Does ipv6 dns entry need also update?

Reply:
The ipv6 tests passed, so I don't think so